cmd/otel-allocator/.golangci.yaml

```diff
@@ -0,0 +1,26 @@
+run:
```

Comment:
It is bad to duplicate this, if the action does not know to read the config from parent dirs, as the binary does, consider to manually run the binary as we do in all the other repos.

Reply:
Okay - I've removed this file.